library/kong

```diff
@@ -2,6 +2,17 @@ Maintainers: Kong Docker Maintainers <support@konghq.com> (@thekonginc)
 GitRepo: https://github.com/Kong/docker-kong.git
 # needs "Constraints" to match "centos" (which this image is "FROM")
 
+Tags: 1.1.0rc2-alpine, 1.1rc2-alpine, 1.1.0rc2, 1.1rc2
+GitCommit: 764155275f26b1447ce34f9255d2564d4fdd08a1
+GitFetch: refs/tags/1.1.0rc2
+Directory: alpine
+
+Tags: 1.1.0rc2-centos, 1.1rc2-centos
+GitCommit: 764155275f26b1447ce34f9255d2564d4fdd08a1
+GitFetch: refs/tags/1.1.0rc2
+Constraints: !aufs
+Directory: centos
+
 Tags: 1.1.0rc1-alpine, 1.1rc1-alpine, 1.1.0rc1, 1.1rc1
```

Comment:
Did you want to drop the `rc1` versions now that you are adding `rc2`?

Reply:
At this time we'll keep it around 👍